src/App.css

```diff
@@ -52,22 +52,39 @@ h1 {
     monospace;
 }
 
-.add-button {
-  padding: 0.5rem 2rem;
-  border: 2px solid;
-  border-radius: 0.25rem;
-  background: #fe9476;
-  color: #263147;
-  text-align: center;
-  text-transform: uppercase;
-  font-family: 'Red Hat Display', 'Segoe UI', Roboto, 'Helvetica Neue', Arial,
-    sans-serif;
-  font-size: 1.33rem;
-  font-weight: 900;
-  line-height: 1.5;
-  cursor: pointer;
+@keyframes bouncing-loader {
+  to {
+    opacity: 0.1;
+    transform: translate3d(0, -16px, 0);
+  }
+}
+
+.bouncing-loader {
+  display: flex;
+  justify-content: center;
+}
+
+.bouncing-loader > div {
+  width: 8px;
+  height: 8px;
+  margin: 3rem 0.2rem;
+  background: #8385aa;
+  border-radius: 50%;
+  animation: bouncing-loader 0.6s infinite alternate;
+}
+
+.bouncing-loader > div:nth-child(2) {
+  animation-delay: 0.2s;
+}
+
+.bouncing-loader > div:nth-child(3) {
+  animation-delay: 0.4s;
 }
```

Comment:
Adding a loader is a nice touch and gives feedback to the user :+1: !

Reply:
Totally, I was very happy to see this feature :)